cg/cli/demultiplex/copy_novaseqx_data.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+import os
+import shutil
+from typing import List, Optional
+
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def is_copied(analysis_directory: Path) -> bool:
```

Comment:
Maybe clear from the caller side context what is copied, but it would be nice with a function name/ doc str describing how this intended to be used as this code could be reused in multiple callers and then it easier to understand if it is safe to reuse

Perhaps:`is_demultipexing_copied`?

Reply:
Yes, these functions could gain from more descriptive names. Sounds good